studio/src/editor-panel.js

```diff
@@ -163,7 +163,8 @@ export default class EditorPanel extends LitElement {
 
     updateFragment(event) {
         const fieldName = event.target.dataset.field;
-        let value = event.target.value || event.detail?.value;
+        let value =
+            event.target.value || event.detail?.value || event.target?.checked;
         value = event.target.multiline ? value?.split(',') : [value ?? ''];
         this.inEdit.updateField(fieldName, value);
```

Comment:
```
updateFragment({target, detail}) {
   const fieldName = target?.dataset.field
   let value = target?.value || detail?.value || target?.checked;
   value = target?.multiline ? value.split(',') : [value ?? ''] ;
} 
```

Reply:
is this really better?
if you need to debug what else is present in event object you will need to find where updateFragment is called and stop there. 
i am good to change is just my preference is to be more clear about what is passed as a function argument

Reply:
ok, then just be consistent too with the ?. :) and we are good